packages/components/molecules/f-rating/src/components/Rating.vue

```diff
@@ -112,10 +113,20 @@ export default {
          *
          * @returns {string}
          */
-        setRatingVariant () {
+        getRatingVariant () {
             return this.isSingleStarVariant
                 ? 'rating-single-star'
                 : 'rating-multi-star';
+        },
+
+        /**
+         * Allow only valid rating display types, so we show
+         * visually hidden text in those cases only.
+         *
+         * @returns {boolean}
+         */
+        hasRatingAvailable () {
+            return this.ratingDisplayType !== 'noRating';
```

Comment:
Had a chat with @kevinrodrigues and we're gonna look to change this to something like `return this.reviewCount > 0;`.

Reply:
Will create a ticket around this and pick it up next. 👍

Reply:
Just a little improvement, that could be added to that ticket too. 😄 

https://github.com/justeat/fozzie-components/blob/ddae7b31ef217ba30af775058149a3c93746329c/packages/components/molecules/f-rating/src/tenants/en-GB.js#L4
https://github.com/justeat/fozzie-components/blob/ddae7b31ef217ba30af775058149a3c93746329c/packages/components/molecules/f-rating/src/components/Rating.vue#L98-L109

Updated to
``` javascript
getRatingDescription () {
  return this.$tc('ratings.starsDescription', this.starRating , {
      rating: this.starRating,
      maxStarRating: this.maxStarRating
  })
}
```

[i18n Docs](https://kazupon.github.io/vue-i18n/guide/pluralization.html#accessing-the-number-via-the-pre-defined-argument)